{"name": "test_log_fail", "status": "passed", "start": 1638028034448, "stop": 1638028034448, "uuid": "bdaf7714-2319-4d7a-b3a9-78e56f816ef8", "historyId": "1c8ef04b5efb530f297c008febc65861", "testCaseId": "7ca6e908bf6479da7772d77349e0c8d1", "fullName": "test_allure_login.TestLogin#test_log_fail", "labels": [{"name": "story", "value": "登录失败"}, {"name": "feature", "value": "登录模块"}, {"name": "suite", "value": "test_allure_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-U9QUJNK"}, {"name": "thread", "value": "18840-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_allure_login"}]}